server/routes/volunteers.test.js

```diff
@@ -4,9 +4,11 @@ const log = require('../logger')
 const server = require('../server')
 const db = require('../db/volunteers')
 const { getMockToken } = require('./mockToken')
+const { decode } = require('../notifications/emailTokens')
 
 jest.mock('../logger')
 jest.mock('../db/volunteers')
+jest.mock('../notifications/emailTokens') // for decode.mockImplementation
```

Comment:
The comments are awesome while you're working on things, but make sure you remove them before we merge into dev!